influxdb3_write/src/paths.rs

```diff
@@ -72,12 +72,14 @@ impl ParquetFilePath {
         db_name: &str,
         table_name: &str,
         partition_key: &str,
+        segment_id: SegmentId,
         file_number: u32,
     ) -> Self {
         let path = ObjPath::from(format!(
-            "dbs/{db_name}/{table_name}/{}/{:010}.{}",
+            "dbs/{db_name}/{table_name}/{}/{:010}_{}.{}",
```

Comment:
Was there a reason you couldn't just use the next file number? If we're going to use the SegmentID for the path I'd prefer it to be:

```suggestion
            "dbs/{db_name}/{table_name}/{}/segment-{:010}/{}.{}",
```

which makes it a file under a segment, rather than part of the file name and denote that it's stored as part of a segment in obj store.

Reply:
The old code had file number, but it was actually a stand in for the segment ID. I"ll change this so that it's a `/` to separate them.